src/tokens/transformation/android/filters/lightColor.js

```diff
@@ -0,0 +1,4 @@
+const { matchLightThemeToken } = require('../../common/tokenFilters')
+
+module.exports = (token) =>
+  matchLightThemeToken(token) || token.name.startsWith('primitive')
```

Comment:
Might be worth adding a comment about why we include the primitive colors here (for the people who aren't familiar with Android, like me :laughing:)